h2integrate/converters/ammonia/ammonia_synloop.py

```diff
@@ -381,10 +381,17 @@ class AmmoniaSynLoopCostModel(CostModelBaseClass):
     """
 
     def setup(self):
-        super().setup()
+        target_cost_year = self.options["plant_config"]["finance_parameters"][
+            "cost_adjustment_parameters"
+        ]["target_dollar_year"]
+        self.options["tech_config"]["model_inputs"]["cost_parameters"].update(
+            {"cost_year": target_cost_year}
```

Comment:
I'm a little confused about why the `cost_year` is being set to the `target_cost_year` here. Can you explain what is going on here?